python/servo/command_base.py

```diff
@@ -205,6 +205,21 @@ def is_macosx():
     return sys.platform == 'darwin'
 
 
+def find_native_win32_python():
+    paths = subprocess.Popen(['where', 'python.exe'],
+                             shell=False, stdout=subprocess.PIPE).stdout.read()
+
+    for p in paths.splitlines():
+        p = p.replace("\\", "/")
+        py_type = subprocess.Popen([p, '-c', 'import sys; print(sys.platform)'],
+                                   shell=False, stdout=subprocess.PIPE).stdout.read()
+        if "win32" in py_type and "/mingw64/bin" not in p:
```

Comment:
We could just check for the presence of `"mingw64"`, instead of going for a path?